toggle-pw.js

```diff
@@ -0,0 +1,21 @@
+export const pw = document.querySelector(".js-pw");
+const pwIcon = document.querySelector(".js-pw-icon");
+
+const hidden = "/images/icon/pw-hidden.png";
+const view = "/images/icon/pw-view.png";
+
+function toggleIcon(e) {
```

Comment:
P2:
안쓰는 인자의 경우 지워주시는 것이 가독성에 더 좋을 것 같습니다.

```suggestion
function toggleIcon(e) {
```